src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (arg) => {
+    if (arg === undefined) {
```

Comment:
It is not necessary to compare exactly with `undefined`, you can simply check whether this argument exists through the logical NOT.
```js
if (!number) {
```

Reply:
Hello Roman, thank you for the comment. I tried that. It's generally like you say, but in this case the ! version won't pass the test if zero has been received as an argument (1)(1)(1)(1)(0)(1)(1)(1)()

Reply:
@Orpheus29 yes, you are right 🔥